solutions/instances/variables.tf

```diff
@@ -40,6 +40,12 @@ variable "prefix" {
 # Log Analysis Variables
 ##############################################################################
 
+variable "log_analysis_provision" {
+  description = "Provision an IBM Cloud Logging instance?"
```

Comment:
Provide a better description instead of having `?` at the end

Reply:
changed